runtime-linux.md

```diff
@@ -0,0 +1,3 @@
+## File descriptors
+By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the container process by the runtime.
+The runtime may leak additional file descriptors into the container process to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
```

Comment:
`container process` should be `container's init process` i think. Do we want to pass extra file descriptors for non-init process ?